guide/installation-and-configuration/general-installation/installation-enterprise.markdown

```diff
@@ -94,10 +94,18 @@ The disk that serves PostgreSQL (/var/cfengine/state/pg) should be able to perfo
 
 **Note** Your storage IOPS specification may be given in 4KiB block size, in which case you would need to divide it by 4 to get the corresponding 16KiB *theoretical maximum*.
 
+### Open file descriptors
+
+The policy server should ideally be able to accept connections from all clients; i.e. to allow at least as many incoming connections as there are clients.
+The system limit for this is controlled by `ulimit -n`; so the parent process from which you bootstrap should, for a server with 5000 hosts, run `uname -n 5000` first.
+You should also add such a `uname -n` command to the script that implements `service cfengine3 start` (and `restart`).
```

Comment:
Not uname.. Also, is it possible to add this to the CFE policy for large hubs (or is it tied to the shell or something else)? @nickanderson would know where it would make sense (probably somewhere in cfe_internal).